cves/CVE-2008-2939.yml

```diff
@@ -53,7 +56,7 @@ fixes:
      note:
 vccs:
   - commit: 5a039b343a02a30d66fd1224f3fbdaa25f476eae
-    note:
```

Comment:
Maybe also add a note for the fix commits.

Reply:
I agree thanks for the heads up! I completely missed that.